trackbasket_be/resources/volunteer.py

```diff
@@ -6,11 +6,31 @@ class Volunteers(Resource):
 
   def get(self,id):
     volunteer = db.session.query(Volunteer).filter_by(volunteer_id=id).first()
-    return {'data': { 'id': 'volunteer', 'attributes': {'id': volunteer.volunteer_id, 'name': volunteer.name, 'phone number': volunteer.phone_number} } }
+    if volunteer is None:
+      return {'data': { 'id': 'volunteer', 'attributes': {'error': "Volunteer not found"} } }, 400
+    else:  
+      return {'data': { 'id': 'volunteer', 'attributes': {'id': volunteer.volunteer_id, 'name': volunteer.name, 'phone number': volunteer.phone_number} } }, 200
 
   def post(self, id):
     request_data = request.json
     volunteer = Volunteer(name=request_data['name'], phone_number=request_data['phone_number'], volunteer_id=id)
     db.session.add(volunteer)
     db.session.commit()
-    return {'volunteer': request_data}
+    return {'volunteer': request_data}, 201
```

Comment:
added the status code for the successful post

Reply:
how about the case if a volunteer tries to create a new account even though they already have one?

Reply:
i like how you specify the status code